src/main/java/core/basesyntax/HelloWorld.java

```diff
@@ -1,8 +1,19 @@
 package core.basesyntax;
 
-/**
- * Feel free to remove this class and create your own.
- */
 public class HelloWorld {
+    private static final int NUMBER_OF_FIGURES = 6;
 
+    public static void main(String[] args) {
```

Comment:
The `main` method should not contain an empty line at the beginning. Please remove the redundant empty line.